smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSymbolProvider.kt

```diff
@@ -363,15 +362,16 @@ class SwiftSymbolProvider(private val model: Model, val swiftSettings: SwiftSett
     // Document: default value can be set to null, true, false, string, numbers, an empty list, or an empty map.
     private fun handleDocumentDefaultValue(literal: String, node: Node, builder: Symbol.Builder): Symbol.Builder {
         var formatString = when {
-            node.isObjectNode -> "\$N.object([:])"
-            node.isArrayNode -> "\$N.array([])"
-            node.isBooleanNode -> "\$N.boolean($literal)"
-            node.isStringNode -> "\$N.string(\"$literal\")"
-            node.isNumberNode -> "\$N.number($literal)"
+            node.isObjectNode -> "[:]"
+            node.isArrayNode -> "[]"
+            node.isBooleanNode -> literal
+            node.isStringNode -> "\"$literal\""
+            node.isNumberNode -> literal
             else -> return builder // no-op
         }
         return builder.defaultValueClosure { writer ->
-            writer.format(formatString, SmithyReadWriteTypes.Document)
+            writer.addImport(SwiftDependency.SMITHY_JSON.target)
+            writer.format(formatString)
```

Comment:
Same as below, once you import SmithyJSON, you can initialize a Document with a bare literal.